changelog/next/features/4806--base64.md

```diff
@@ -0,0 +1,2 @@
+Two new functions `encode_base64` and `decode_base64` have been added to easily
+Base64 encode and decode blobs and strings.
```

Comment:
Please prefer active voice and present tense in documentation and changelog entries.